purchase_post_code_propose_address_sst/i18n/ja.po

```diff
@@ -1,97 +1,102 @@
 # Translation of Odoo Server.
 # This file contains the translation of the following modules:
-# * purchase_post_code_propose_address_sst
+# 	* purchase_post_code_propose_address_sst
 #
 msgid ""
 msgstr ""
-"Project-Id-Version: Odoo Server 11.0+e\n"
+"Project-Id-Version: Odoo Server 15.0\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2019-04-26 07:18+0000\n"
-"PO-Revision-Date: 2019-04-26 07:18+0000\n"
-"Last-Translator: <>\n"
+"POT-Creation-Date: 2023-10-27 03:54+0000\n"
+"PO-Revision-Date: 2023-10-27 03:54+0000\n"
+"Last-Translator: \n"
 "Language-Team: \n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: \n"
 "Plural-Forms: \n"
 
 #. module: purchase_post_code_propose_address_sst
-#: model:ir.ui.view,arch_db:purchase_post_code_propose_address_sst.purchase_order_form
+#: model_terms:ir.ui.view,arch_db:purchase_post_code_propose_address_sst.purchase_order_form
+msgid "<span class=\"oe_read_only\">〒</span>"
+msgstr ""
+
+#. module: purchase_post_code_propose_address_sst
+#: model_terms:ir.ui.view,arch_db:purchase_post_code_propose_address_sst.purchase_order_form
 msgid "Address"
-msgstr "アドレス"
+msgstr ""
```

Comment:
@AungKoKoLin1997 Can you please revive the translation where the original string hasn't changed.